### ISSUE TITLE
Display contents of README.clear later/last for autospec

### ISSUE THREAD

**Issue by mdhorn** [clearlinux/distribution#1512]
Currently, if a README.clear file exists in the package repository, it is displayed via the autospec.py file. There is considerable output after this file is printed, and it generally scrolls off the visible display.

As one of the main reasons for this file is to highlight something to the package builder, it would be preferable for this content to be later in the flow, if not last. 

Suggest moving the displaying of this content from autospec.py to being in the common Makefile around the same place as the "for-review.txt" note.

**Comment by phmccarty** [comment]
I agree that the output needs to be improved. Likely the common tooling should print the message, but it needs to compete for attention with the `for-review.txt` message... I will explore various options.
